### DISCUSSION TITLE
天翼云盘webdav上传超过50M的文件提示失败

### DISCUSSION THREAD

**Discussion by 404gods** [AlistGo/alist#434] [Q&A] [answered]
我在群晖docker中搭建了alist，并正常设置了webdav。
我发现用群晖自带的cloud sync通过webdav上传文件到189天翼云盘时，超过50M的文件全部上传失败，提示服务器错误或由远程服务器锁定。
小文件上传都正常没有错误发生。我的189账号是黄金会员不会有50M大小的上传限制，我也在网页中尝试了上传大文件都是正常的。
请问这个问题怎么办？
![捕获](https://user-images.githubusercontent.com/50573302/150716161-bbbb0751-920a-4b7e-89cd-2fcd99db8c7b.PNG)

**Comment by xhofe** [comment, marked as answer]
可能那个post multipart/form-data的接口做了限制了，暂时还未实现chunk上传

**Reply by 404gods** [reply]
感谢回复，要是没这个问题就能把天翼云盘当作备份空间了，真可惜

**Reply by xhofe** [reply]
是官方网页端还是alist网页端？如果是alist网页端可以上传的话 那接口就没有限制，可以看看日志中有没有报什么错误。

但是通过目前的post multipart/formdata会需要很大的内存，上传大文件容易oom。

**Reply by 404gods** [reply]
谢谢答复，今天升级了最新的docker镜像重新安装（v2.0.8-fix），可以正常使用cloud sync通过webdav进行189网盘备份了。

**Reply by 404gods** [reply]
最新回复，本来可以正常备份，晚上发现提示错误，189网盘文件夹里的备份文件重复上传达几百份……放弃了
